ChromecastSampleApp/app/build.gradle

```diff
@@ -7,7 +7,7 @@ android {
     defaultConfig {
         applicationId "com.ooyala.sample.ChromecastSampleApp"
         minSdkVersion 14
-        targetSdkVersion 23
+        targetSdkVersion 21
```

Comment:
I was running into compilation error in one of the mac minis( some combination of Android build tools and targetSdkVersion ) was failing . 
I tried again now with targetSdkVersion 23 and it works just fine . 
Shall i change it back to 23 ?

Shall i change the targetSdkVersion back to 23 ?